client/src/saga/county/acvrUploadSaga.ts

```diff
@@ -1,4 +1,4 @@
-import { takeEvery } from 'redux-saga/effects';
+import { takeLatest } from 'redux-saga/effects';
```

Comment:
Actually, I wanted the behavior described in `takeLatest` where if multiple of these actions that are listened for here get to be in flight at once, the saga middleware will cancel the earlier ones and only process the "latest" one, thus showing only one of that type of notice even if the action was received multiple times. For a saga that does nothing but display a notice, I think this would be a hard condition to meet, it just felt closer to what we want.

Reply:
That makes sense.